packages/discord.js/src/structures/interfaces/Collector.js

```diff
@@ -97,6 +97,20 @@ class Collector extends EventEmitter {
     if (options.idle) this._idletimeout = setTimeout(() => this.stop('idle'), options.idle).unref();
   }
 
+  /**
+   * The timestamp at which this collector last collected an item
+   * @type {?number}
+   */
+  lastCollectedTimestamp = null;
```

Comment:
All the properties are assigned in the constructor, not as fields, this should be moved inside the constructor.

Reply:
don't really get why but I'll change this